packages/testkit-backend/src/request-handlers.js

```diff
@@ -43,16 +42,18 @@ export function NewDriver (context, data, wire) {
   }
   const resolver = resolverRegistered
     ? address =>
-      new Promise((resolve, reject) => {
-        const id = context.addResolverRequest(resolve, reject)
-        wire.writeResponse(responses.ResolverResolutionRequired({ id, address }))
-      })
+        new Promise((resolve, reject) => {
+          const id = context.addResolverRequest(resolve, reject)
+          wire.writeResponse(responses.ResolverResolutionRequired({ id, address }))
+        })
     : undefined
+
+  const envLogLevel = typeof process === 'undefined' ? undefined : process.env.LOG_LEVEL
   const config = {
     userAgent,
     resolver,
     useBigInt: true,
-    logging: neo4j.logging.console(process.env.LOG_LEVEL || context.logLevel)
+    logging: neo4j.logging.console(envLogLevel || context.logLevel)
```

Comment:
Does that mean there's no way to configure the log level of drivers controlled by Deno backend?

Reply:
Not yet.